package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
There are a lot of changes in the lock file because I deleted it and recreated it from scratch. This gets rid of past dependencies/versions and rebuilds them all.